src/modules/clark/learning-object-module/learning-objects.routes.ts

```diff
@@ -241,7 +241,7 @@ export const LEARNING_OBJECTS_ROUTES: ProxyRoute[] = [
      */
     {
         method: HTTPMethod.PATCH,
-        path: "/users/:username/learning-objects/:id/relevancy-check",
+        path: "/learning-objects/:id/relevancy-check",
         auth: true,
```

Comment:
We need to add a target.

Reply:
What do you mean by this?

Reply:
There is an optional key called target that redirects to the default target. Currently the default target is learning object service URL in the router.ts file for the module.

```
target = envConfig.getClarkServiceUri()
```

or something like that ^^ will redirect this route to clark service rather than learning object service.